Makefile

```diff
@@ -1,21 +1,32 @@
-SKIP_SQUASH?=0
-VERSIONS="2.4.41"
+OPENLDAP_VERSION := $(shell git describe --tags --always --dirty)
```

Comment:
why the `shell`?

Reply:
I wanted to version the repository and follow along with that tag in the images. Not a good idea?

E.g. someone wants to recover mrogers950/origin-openldap-test. That person would need to look up git tag 2.4.46 of this repository.

Reply:
This is also kind of the problem that I framed in the description of the PR: those versions are not pinned. So actually the mrogers950/openldap image uses a slightly higher version as does the openldap2441 one.

Reply:
yeah but why call "shell"?

Reply:
This is a Makefile, hence `$(shell ` syntax (https://www.gnu.org/software/make/manual/html_node/Shell-Function.html), it's not a bash script.

Reply:
oh, my bad, I did not realize it's Makefile :facepalm:

Reply:
I hardcoded now the image tag by `image OS` and not by `git tag`.